packages/core/src/App/Components/Layout/Header/platform-dropdown.jsx

```diff
@@ -35,6 +35,49 @@ const PlatformDropdown = ({ app_routing_history, closeDrawer, platform_config })
             closeDrawer();
         }
     };
+    const { is_pre_appstore } = React.useContext(PlatformContext);
+    const getPlatformDropdownContent = platform => {
+        let platform_dropdown_content;
+        if (is_pre_appstore) {
+            if (
+                platform.name !== getPlatformSettings('mt5').name &&
+                platform.name !== getPlatformSettings('dxtrade').name
+            ) {
+                platform_dropdown_content = platform.link_to ? (
+                    <BinaryLink
+                        to={platform.link_to}
+                        // This is here because in routes-config it needs to have children, but not in menu
+                        exact={platform.link_to === routes.trade}
+                        className='platform-dropdown__list-platform'
+                        isActive={() => getActivePlatform(app_routing_history) === platform.name}
+                    >
+                        <PlatformBox platform={platform} />
```

Comment:
this line has been duplicated 4 times, by right you need to store it in a variable and reuse it for a shorter piece of code